lib/measures/tbd/measure.rb

```diff
@@ -218,36 +240,59 @@ def arguments(model = nil)
     args << gen_kiva
 
     arg = "gen_kiva_force"
-    dsc = "Overwrites 'ground' boundary conditions as 'foundation' before "    \
+    dsc = "Overwrites 'ground' boundary conditions as 'foundation' before "\
           "generating Kiva inputs (recommended)."
     kiva_force = OpenStudio::Measure::OSArgument.makeBoolArgument(arg, false)
     kiva_force.setDisplayName("Force-generate Kiva inputs")
     kiva_force.setDescription(dsc)
     kiva_force.setDefaultValue(false)
     args << kiva_force
 
-    return args
+    args
   end
 
+  ##
+  # Runs the OpenStudio TBD Measure.
+  #
+  # @param mdl [OpenStudio::Model::Model] a model
+  # @param runner [OpenStudio::Measure::OSRunner] Measure runner
+  # @param [OpenStudio::Measure::OSArgumentVector] args Measure argument list
+  # @option args [Bool] :alter (true) irreversibly applies changes to the model
+  # @option args [#to_f] :sub_tol (OSut::TOL) proximity tolerance between edges
+  # @option args [Bool] :load_tbd (false) whether to load a TBD JSON input file
+  # @option args [#to_s] :option ("poor (BETBG)") selected PSI set
+  # @option args [Bool] :write_tbd_json (false) whether to output a JSON file
+  # @option args [#to_s] :wall_option wall(s) construction to uprate
+  # @option args [#to_s] :roof_option roof(s) construction to uprate
+  # @option args [#to_s] :floor_option floor(s) construction to uprate
+  # @option args [#to_f] :wall_ut (0.210) target wall Ut, in (W/m2•K)
+  # @option args [#to_f] :roof_ut (0.138) target roof Ut, in (W/m2•K)
+  # @option args [#to_f] :floor_ut (0.162) target floor Ut, in (W/m2•K)
+  # @option args [Bool] :gen_UA_report (false) whether to generate a UA' report
+  # @option args [#to_s] :ua_reference ("code (Quebec)") UA' ruleset
+  # @option args [Bool] :gen_kiva (false) whether to generate KIVA inputs
+  # @option args [Bool] :gen_kiva_force (false) whether to force KIVA inputs
+  #
+  # @return [Bool] whether TBD Measure is successful
```

Comment:
In an effort to ensure 100% YARD-ocumentation, TBD method parameters and return objects are _better_ documented, notably for input/output hashes. This includes run-of-the-mill _measure.rb_ methods. TBD dependencies like OSut are documented elsewhere.